src/shared-components/immersiveModal/style.ts

```diff
@@ -37,7 +36,7 @@ const Overlay = styled.div`
   }
 `;
 
-const CrossIconContainer = styled.button`
+const CrossIconButton = styled.button<{ showDesktopHeaderBar?: boolean }>`
```

Comment:
Renamed to reflect that this is, in fact, a button. `DialogModal` still uses a `div` and so I kept the name container there.